.github/workflows/docker.yaml

```diff
@@ -0,0 +1,78 @@
+name: DOCKER
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      slurm:
+        image: autosubmit/slurm-openssh-container:25-05-0-1
+        ports:
+          - 2222:2222
+        options: --rm -it --cgroupns=host --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw -h slurmctld --name slurm-container
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Copy access key
+        run: |
+          docker cp slurm-container:/root/.ssh/container_root_pubkey /tmp/container_root_pubkey
+          chmod 600 /tmp/container_root_pubkey
+
+      - name: configure host
+        run: |
+          sudo bash -c 'mkdir /home/runner/.ssh/'
+          sudo bash -c 'echo -e $"Host localDocker\n  HostName localhost\n  User root\nStrictHostKeyChecking no\n UserKnownHostsFile /dev/null\n  IdentityFile /tmp/container_root_pubkey\n  Port 2222\n   ForwardX11 yes" >> /home/runner/.ssh/config'
+
+      - name: ssh access
+        run: |
+          cat /home/runner/.ssh/config
+          cat /tmp/container_root_pubkey
+
+      - name: SSH to SLURM environment
+        run: |
+          ssh localDocker -p 2222 -i /tmp/container_root_pubkey sinfo
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install system dependencies
+        run: sudo apt-get install -y curl git graphviz rsync
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip packaging setuptools twine
+          pip install --upgrade -e .[all]
+
+      - name: Install test dependencies
+        run: |
+          pip install pytest pytest-cov pytest-xdist
+
+      - name: Set up Git
+        run: |
+          # From: https://stackoverflow.com/questions/62960533/how-to-use-git-commands-during-a-github-action
+          # Set up dummy configuration for integration tests.
+          git --version
+          git config --global user.email "test@bsc.es"
+          git config --global user.name "GitHub Actions"
+
+      - name: Integration tests
+        run: |
+          pytest \
+            --cov=autosubmit --cov-config=.coveragerc \
+            --cov-report=xml:test/coverage.xml --cov-append \
+            test/integration/platforms/test_slurmplatform.py \
+            -m ''
```

Comment:
We need to create a marker for Slurm. Then the integration tests should pass without any modification, and here you just set something like `-m 'slurm'`.